toolchain/sem_ir/inst_kind.h

```diff
@@ -82,6 +83,12 @@ class InstKind : public CARBON_ENUM_BASE(InstKind) {
       TerminatorKind terminator_kind = TerminatorKind::NotTerminator) const
       -> Definition<TypedNodeId>;
 
+  // As above, but for instructions which are not lowered; `Define(ir_name)`
+  // should be used instead of specifying `is_lowered = true`.
+  template <typename TypedNodeId>
+  constexpr auto Define(llvm::StringLiteral ir_name, bool is_lowered) const
+      -> Definition<TypedNodeId>;
```

Comment:
I would prefer that we use an enum instead of the `bool` parameter plus parameter name comments.

Reply:
Is that including in #4119, you would prefer `.is_lowered = IsLowered::True`?

Reply:
Per discussion, let's use #4119 to settle final state.